isofit/test/test_cli_machinery.py

```diff
@@ -1,19 +1,38 @@
 """High-level tests to ensure the CLI is constructed properly."""
 
+import os
 import subprocess as sp
 import sys
+from pathlib import Path
 
 import pytest
 from click.testing import CliRunner
 
 import isofit
 from isofit import __main__
 
+# Determine if 'isofit' is installed via '$PYTHONPATH'. To do this, check to see
+# if '$PYTHONPATH' is set, and search for 'isofit' at each location.
+ISOFIT_ABSPATH = Path(isofit.__file__).parent.absolute()
+IS_INSTALLED_VIA_PYTHONPATH = False
+if "PYTHONPATH" in os.environ:
+    for p in os.environ["PYTHONPATH"].split(os.pathsep):
+        potential_isofit_path = Path(p).absolute() / isofit.__name__
+        if potential_isofit_path == ISOFIT_ABSPATH:
+            IS_INSTALLED_VIA_PYTHONPATH = True
+            break
```

Comment:
As far as I can tell this is a robust check to determine if a package is installed via `PYTHONPATH`. It is required to determine if a test should be run against `$ isofit` or `$ python3 -m isofit`.